integrationTests/chainSimulator/bridge/simpleEsdt_test.go

```diff
@@ -0,0 +1,446 @@
+		NumNodesWaitingListMeta:  0,
+		NumNodesWaitingListShard: 0,
+		AlterConfigsFunction: func(cfg *config.Configs) {
+			cfg.GeneralConfig.VirtualMachine.Execution.TransferAndExecuteByUserAddresses = []string{whiteListedAddress}
+			cfg.EpochConfig.EnableEpochs.DynamicESDTEnableEpoch = 0
+		},
+	})
+	require.Nil(t, err)
+	require.NotNil(t, cs)
+
+	defer cs.Close()
+
+	err = cs.GenerateBlocksUntilEpochIsReached(4)
+	require.Nil(t, err)
+
+	nodeHandler := cs.GetNodeHandler(0)
+	systemContractDeploy := chainSim.GetSysContactDeployAddressBytes(t, nodeHandler)
+
+	initialAddress := "erd1l6xt0rqlyzw56a3k8xwwshq2dcjwy3q9cppucvqsmdyw8r98dz3sae0kxl"
+	initialAddrBytes, err := nodeHandler.GetCoreComponents().AddressPubKeyConverter().Decode(initialAddress)
+	err = cs.SetStateMultiple([]*dtos.AddressState{
+		{
+			Address: initialAddress,
+			Balance: "10000000000000000000000",
+		},
+		{
+			Address: esdtSystemAccount, // init sys account
+		},
+	})
+	require.Nil(t, err)
+	nonce := uint64(0)
+
+	err = cs.GenerateBlocks(1)
+	require.Nil(t, err)
+
+	contractAddress := chainSim.DeployContract(t, cs, initialAddrBytes, &nonce, systemContractDeploy, "", simpleEsdtTestWasmPath)
+
+	txResult := chainSim.SendTransaction(t, cs, initialAddrBytes, &nonce, contractAddress, chainSim.ZeroValue, txData, uint64(20000000))
+	chainSim.RequireSignalError(t, txResult, expectedErr)
+}
+
+func TestChainSimulator_CreateAndBurnAllEsdtTypes(t *testing.T) {
+	if testing.Short() {
+		t.Skip("this is not a short test")
+	}
+
+	roundsPerEpoch := core.OptionalUint64{
+		HasValue: true,
+		Value:    20,
+	}
+
+	whiteListedAddress := "erd1qqqqqqqqqqqqqpgqmzzm05jeav6d5qvna0q2pmcllelkz8xddz3syjszx5"
+	cs, err := chainSimulator.NewChainSimulator(chainSimulator.ArgsChainSimulator{
+		BypassTxSignatureCheck:   true,
+		TempDir:                  t.TempDir(),
+		PathToInitialConfig:      defaultPathToInitialConfig,
+		NumOfShards:              3,
+		GenesisTimestamp:         time.Now().Unix(),
+		RoundDurationInMillis:    uint64(6000),
+		RoundsPerEpoch:           roundsPerEpoch,
+		ApiInterface:             api.NewNoApiInterface(),
+		MinNodesPerShard:         3,
+		MetaChainMinNodes:        3,
+		NumNodesWaitingListMeta:  0,
+		NumNodesWaitingListShard: 0,
+		AlterConfigsFunction: func(cfg *config.Configs) {
+			cfg.GeneralConfig.VirtualMachine.Execution.TransferAndExecuteByUserAddresses = []string{whiteListedAddress}
+			cfg.EpochConfig.EnableEpochs.DynamicESDTEnableEpoch = 0
+		},
+	})
+	require.Nil(t, err)
+	require.NotNil(t, cs)
+
+	defer cs.Close()
+
+	err = cs.GenerateBlocksUntilEpochIsReached(4)
+	require.Nil(t, err)
+
+	nodeHandler := cs.GetNodeHandler(0)
+	systemContractDeploy := chainSim.GetSysContactDeployAddressBytes(t, nodeHandler)
+
+	initialAddress := "erd1l6xt0rqlyzw56a3k8xwwshq2dcjwy3q9cppucvqsmdyw8r98dz3sae0kxl"
+	initialAddrBytes, err := nodeHandler.GetCoreComponents().AddressPubKeyConverter().Decode(initialAddress)
+	err = cs.SetStateMultiple([]*dtos.AddressState{
```

Comment:
Same questions for this test as above

Reply:
Yes, this was a recent change in chain simulator and need to generate 1 block after set state